### ISSUE TITLE
Compile to rust code?

### ISSUE THREAD

**Issue by IvanTurgenev** [SilverWingedSeraph/sbic#1]

**Comment by NoraCodes** [comment]
Could you expand a little more? What's the goal?

**Comment by IvanTurgenev** [comment]
instead of interpreting run native meaning run faster?

**Comment by NoraCodes** [comment]
I don't think it would be much faster unless the compiler did optimizations. In any case, my use case is as a genetic medium for machine learning so total time from source code to first run is my main concern. Compilation would only increase that.

**Comment by IvanTurgenev** [comment]
ok, or compile the final?
